schema/nanotdf/README.md

```diff
@@ -321,7 +325,9 @@ The following are the available values:
 |---------|---------------------------|
 | `0x00`  | `http`                    |
 | `0x01`  | `https`                   |
-| `0x02`  | unreserved                |
+| `0x21`  | `https` w/identifier 2B   |
+| `0x81`  | `https` w/identifier 8B   |
+| `0xf1`  | `https` w/identifier 32B  |
 | `0xff`  | Shared Resource Directory |
```

Comment:
```suggestion
|Bits 3-0 | Protocol Enum Value |
|-------|------|
| `0x0`  | `http`   |
| `0x1`  | `https`  |
| `0x2`  | unreserved  |
| `0xf`  | Shared Resource Directory  |

| Bits 7-4 | Meaning for KAS key lookup |
|-------|------|
| `0x0`  | No Key Identifier   |
| `0x1`  | 2 Byte Identifier |
| `0x2`  | 8 Byte Identifier  |
| `0x3`  | 32 Byte Identifier |
```

This allows different interpretation of the high 4 bits in different contexts (non-kas-key lookup)